ga4gh/backend.py

```diff
@@ -90,19 +91,18 @@ def runSearchRequest(
             request.pageSize = self._defaultPageSize
         if request.pageSize <= 0:
             raise exceptions.BadPageSizeException(request.pageSize)
-        pageList = []
+        response = protocol.SearchResponseBuffer(
```

Comment:
Well, we're  assuming a certain structure of the response object in the `SearchResponseBuilder`, which the `SearchXResponse` class makes more explicit. Not much more explicit, mind, because we have to pass the `pageListName` variable around. Our lives would have been easier and we would have much cleaner code if all the `SearchXResponse` classes had the same structure and used `values` (or something) instead of `variants`, `callSets`, etc. etc. But they don't, so we need these ugly hacks.

We still need the class in any case though, because we want to validate against it: we should be able to verify that our results conform to the schema.